src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt

```diff
@@ -911,7 +912,7 @@ object HighwayTools : Module() {
                         append("\n    §9> §7Coordinate: §a${startingBlockPos.z}§r")
                     }
                 }
-
+                if (startingBlockPos.y >= 118 && startingBlockPos.x < 120) append("\n    §9> §7You should move to Y 120 to build proper highways")
```

Comment:
The second condition should be .y not .x, also you may want to replace the check with:
`if (startingBlockPos.y != 119)` (or whatever the correct block height is)

Reply:
Ur right, ima change thanks for showing me my mistake 👍

Reply:
The thing is that you do use different Y sometimes to build highways so I don't want to do it !- 120, cus there is no reason for you to build highways at Y 118-119 but there is for 117 and lower.